install.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+DIRECTORY="fcc-python-vagrant"
+
+echo "This script requires superuser access to install apt packages."
+echo "You will be prompted for your password by sudo."
+
+# clear any previous sudo permission
+sudo -k
+
+# run inside sudo
+sudo sh <<SCRIPT
```

Comment:
If I enter three incorrect passwords, the script should completely error out. In its current state, it's running everything after `SCRIPT` tag.